cmd/admin-handlers.go

```diff
@@ -2612,7 +2612,8 @@ func getClusterMetaInfo(ctx context.Context) []byte {
 	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
 	defer cancel()
 
-	resultCh := make(chan madmin.ClusterRegistrationInfo)
+	resultCh := make(chan madmin.ClusterRegistrationInfo,1)
```

Comment:
this is a wrong fix for this issue,  what we need to do is `defer close(resultCh)`